lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -65,25 +79,20 @@ def code
       end
 
       # REQUIRED - This is the http status code.
-      # unless you've specified that you want the status code to be something other
-      # then 400 explicitly it will default to 400. IT WILL NOT DEFAULT to whatever
-      # was provided by the backend service, because the backend service response
-      # might not always be relevant
       def status
-        i18n_data[:status].presence || 400
+        i18n_data[:status] || 400
```

Comment:
Agree that we shouldn't just pass the downstream status, it's almost always 500 from my experience and it's not us it's them :) IMO 502 would be a better default than 400.